### PR TITLE
[Feature][Map view]

### PR DESCRIPTION
**Pull request by pstoebenau**
### Summary
- A map view was added to each cell that contained location information. The view is presented as a modal.

### Test Plan
- Opening and closing modal
- Passing map information to map view
- Zooming within the modal

### Screenshots/GIFs
Screen Shot 2020-01-02 at 11 36 41 AM
Screen Shot 2020-01-02 at 11 37 02 AM

### CODE REVIEW

File: Source/View Controller/MapViewController.swift
Comment (lloydoad) [inline Source/View Controller/MapViewController.swift]: This should be at the bottom

File: Source/View Controller/ScheduleTableViewController.swift
Comment (lloydoad) [inline Source/View Controller/ScheduleTableViewController.swift]: Is this being used for anything?

File: Source/View Controller/ScheduleTableViewController.swift
Comment (lloydoad) [inline Source/View Controller/ScheduleTableViewController.swift]: Good looking out. You can take out the "TODO" part of this comment because it will trigger a warning

File: Source/View Controller/MapViewController.swift
Comment (lloydoad) [inline Source/View Controller/MapViewController.swift:45]: Great abstraction!

File: Source/View Controller/WorkshopTableViewController.swift
Comment (lloydoad) [inline Source/View Controller/WorkshopTableViewController.swift:79]: Multi-guard statements should be on new lines to make it easier to read. Makes it easier to see what ends where

File: Source/View Model/MapViewControllerModel.swift
Comment (lloydoad) [inline Source/View Model/MapViewControllerModel.swift]: Is this being used for anything?

File: Source/View Controller/SponsorsTableViewController.swift
Comment (lloydoad) [inline Source/View Controller/SponsorsTableViewController.swift]: DestinationID should be a static variable in `MapViewController` to be it easier to edit in the future. If it's defined at multiple places, it would be easy to miss one of them later.

Comment (lloydoad) [reply Source/View Controller/SponsorsTableViewController.swift]: Also sponsors doesn't have any map feature. Its only for schedule and workshops.

**Review by lloydoad — CHANGES_REQUESTED** [review]
Great job overall. Noticed a few things here and there with styling. Also to reiterate, the map feature is only for workshops and schedule; sponsor cells shouldn't open a map.

File: Source/View Controller/ScheduleTableViewController.swift
Comment (pstoebenau) [reply Source/View Controller/ScheduleTableViewController.swift]: no

File: Source/View Controller/ScheduleTableViewController.swift
Comment (pstoebenau) [reply Source/View Controller/ScheduleTableViewController.swift]: I separated the TODO into TO DO. I was also confused as to why swift lint does that.

File: Source/View Controller/MapViewController.swift
Comment (pstoebenau) [reply Source/View Controller/MapViewController.swift:45]: thx

**Review by lloydoad — CHANGES_REQUESTED** [review]
There are some comments from the last review not addressed. There might be a merge conflict not fixed properly because PR isn't building. You can click the `checks` tab for more info on that.

**Comment by pstoebenau** [comment]
I'm not sure which comments you are talking about. I'll make sure to respond to each with what I did now. In addition I'm checking out the build problem.

File: Source/View Controller/MapViewController.swift
Comment (pstoebenau) [reply Source/View Controller/MapViewController.swift]: I moved this to the bottom as you suggested.

File: Source/View Controller/ScheduleTableViewController.swift
Comment (pstoebenau) [reply Source/View Controller/ScheduleTableViewController.swift]: I removed this code. If you were curious it was for the floating button that I previously implemented for accessing the map view.

File: Source/View Model/MapViewControllerModel.swift
Comment (pstoebenau) [reply Source/View Model/MapViewControllerModel.swift]: This is not necessary and thanks for pointing this out. I deleted it.

File: Source/View Controller/WorkshopTableViewController.swift
Comment (pstoebenau) [reply Source/View Controller/WorkshopTableViewController.swift:79]: I did the suggested edits although I'm not sure that this is what you wanted. Did you mean something different?

File: Source/View Controller/SponsorsTableViewController.swift
Comment (pstoebenau) [reply Source/View Controller/SponsorsTableViewController.swift]: I think I misunderstood initially. I initially thought that having a static variable to denote the identifier would be harder change later on. I'll make sure to implement this after I fix the build failure due to the merge.

**Comment by lloydoad** [comment]
Helped out with the build issue. Will approve if project passes tests

**Review by lloydoad — DISMISSED** [review]

**Comment by pstoebenau** [comment]
I think everything is ready at this point.

File: Source/View Controller/MapViewController.swift
Comment (lloydoad) [inline Source/View Controller/MapViewController.swift]: This should be at the bottom of the view controller

File: Source/View Controller/ScheduleTableViewController.swift
Comment (lloydoad) [inline Source/View Controller/ScheduleTableViewController.swift]: This code should be removed if its not being used

**Review by lloydoad — CHANGES_REQUESTED** [review]

**Review by lloydoad — APPROVED** [review]

**Pull request merged**
